src/InstrumentationEngine/ProfilerManager.h

```diff
@@ -244,20 +251,29 @@ namespace MicrosoftInstrumentationEngine
         DWORD CalculateEventMask(DWORD dwAdditionalFlags);
 
         HRESULT InitializeCore(
-            _In_ IUnknown* pCorProfilerInfoUnk,
-            _In_ const vector<CComPtr<CConfigurationSource>>& configSources
+            _In_ IUnknown* pCorProfilerInfoUnk
             );
 
         // The CLR doesn't initialize com before calling the profiler, and the profiler manager cannot do so itself
         // as that would screw up the com state for the application thread. This thread allows the profiler manager
         // to co create a free threaded version of msxml on a thread that it owns to avoid this.
         //
-        // lpParameter points to the this pointer so m_configFilePaths can be accessed. This is safe because the thread proc
+        // lpParameter points to the this pointer so m_configSources can be accessed. This is safe because the thread proc
         // blocks until this is complete.
         static DWORD WINAPI InstrumentationMethodThreadProc(
             _In_  LPVOID lpParameter
             );
 
+        // lpParameter points to the this pointer so m_wszConfigXml can be accessed. This is safe because the thread proc
+        // blocks until this is complete.
+        static DWORD WINAPI ParseAttachConfigurationThreadProc(
+            _In_ LPVOID lpParameter
+            );
+
+        static HRESULT ParseSettingsConfigurationNode(
```

Comment:
>ParseSettingsConfigurationNode [](start = 23, length = 30)

Maybe add a brief comment with a small example of what this method expects the XML node to look like.